models/cms_hcc/final/cms_hcc__patient_risk_scores.sql

```diff
@@ -24,6 +24,18 @@ with seed_adjustment_rates as (
 
 )
 
+, member_months as (
+
+    select
+        patient_id
+        , cast(substr(year_month, 1, 4) as integer) as eligible_year
+        , COUNT(1) as member_months
+    from  {{ ref('financial_pmpm__member_months') }}
```

Comment:
One of our design patterns is to include an ephemeral staging model for every data mart. You can think of this as an "input layer" for the data mart. This helps with documentation and mapping. 

You can see in the [staging](https://github.com/tuva-health/tuva/tree/main/models/cms_hcc/staging) folder for this mart that we are referencing models from Core. Can you add a new staging model for `financial_pmpm__member_months`? You can then replace your reference in this CTE with the staging model. I would name it something like `cms_hcc__stg_financial_pmpm__member_months`.